cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -674,7 +676,7 @@ io.cdap.cdap.runtime.spi.provisioner.ClusterStatus getClusterStatus(String name)
         if (page.getPageElementCount() > 1) {
           // shouldn't be possible
           LOG.warn("Multiple create operations found for cluster {}, may not be able to find the failure message.",
-                   name);
+            name);
```

Comment:
indentation. Method arguments should align with the previous line

Reply:
Used the setting you told me about earlier and resolved.